### ISSUE TITLE
Fabrication [fabricate the parts necessary for the turbidity sensor and the endoscope sensor]

### ISSUE THREAD

**Issue by lois-lee** [AguaClara/sensor_dev#3]
[fabricate the parts necessary for the turbidity sensor and the endoscope sensor]

**Comment by lois-lee** [comment]
We've been working so far on waterproofing the turbidity sensor by sealing PVC to the top of the sensor and then using more flexible tubing for the wiring.

For the Borescope, we've realized that the design will depend heavily on a lot of variables which have to be tested, so we have to test it in an environment similar to the floc hopper and then begin fabrication.

**Comment by hs649** [comment]
The prototype of the waterproof casing is near finished. 
The variables include the reflectivity of the container and of the sediment. It would be helpful to know these characteristics of the floc hopper, but for now our goal is to gauge the consistency of the borescope feedback while those factors are eliminated as much as possible, such as with a very large and/or matte container and dark sediment.

**Comment by HannahSi** [comment]
Forgot to post this last week. We finished fabrication of the turbidity sensor casing, shown in the picture attached. For some reason though, when we tested the encased and uncased sensors on Thursday, neither seemed to working properly--their values didn't fluctuate enough with changes in light. We're going to test and troubleshoot the uncased turbidity sensor first to see if it's poor connections or something else.

![20180228_175651](https://user-images.githubusercontent.com/35944490/37052334-fd172f1a-2146-11e8-8e65-ad6f1b3144ca.jpg)
![20180301_130502](https://user-images.githubusercontent.com/35944490/37052298-e4a05754-2146-11e8-8544-8765cb546d35.jpg)
